.github/workflows/label-checker.yml

```diff
@@ -0,0 +1,18 @@
+name: Label Check
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled, edited]
+jobs:
+  check_label:
+    runs-on: ubuntu-latest
+    name: Check PR Type label is set
+    steps:
+      - name: "Failed to find proper PR Type label. Please add one of the following: 'New Feature', 'Enhancement', or 'Bug-Fix'"
+        if: |
+          !contains(github.event.pull_request.labels.*.name, 'New Feature') &&
```

Comment:
Shouldn't this be "or" statements?

Reply:
if does NOT have blah and does NOT have blah and does NOT have blah. (Making sure all 3 labels are missing.

Reply:
I had to think about it a bit, but I think you're right since we're negating contains for the "when to fail"